### PR TITLE
scripts: Re-add tags for hypervisor config script

### LINKED ISSUE #13: Fix hypervisor config script
#12 had a couple of issues:

- doesn't include the `security:` tag for the static config option.
- makes use of unset variables (`${qemu_version_major}` and `${qemu_version_minor}`).

### PR DESCRIPTION
**Pull request by jodh-intel**
PR #12 inadvertently removed the required tags for two of the qemu
config options.

Partially fixes #13.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
Hi @chavafg - where are `${qemu_version_major}` and `${qemu_version_minor}` supposed to be defined?

**Comment by chavafg** [comment]
Ups, I completely missed to add these lines:
https://github.com/kata-containers/qemu/blob/stable-2.11/configure.sh#L34-L40

I can open a PR to add them, or if it does not bother you, you can add them to this one.

**Comment by jodh-intel** [comment]
Hi @chavafg - no problem. I'll add them on this PR...

**Comment by jodh-intel** [comment]
Hi @chavafg - branch updated.

**Comment by chavafg** [comment]
lgtm
Thanks @jodh-intel

File: scripts/configure-hypervisor.sh [hunk 1]
```diff
@@ -92,10 +92,15 @@ check_tag()
 	local tag="$1"
 	local entry="$2"
 
+	[ -z "$tag" ] && die "no tag for entry '$entry'"
+	[ -z "$entry" ] && echo -e "\n" && die "no entry for tag '$tag'"
```
Comment (devimc) [inline scripts/configure-hypervisor.sh]: why `echo -e "\n"` ?

File: scripts/configure-hypervisor.sh [hunk 2]
```diff
@@ -92,10 +92,15 @@ check_tag()
...
+
 	value="${recognised_tags[$tag]}"
 
+	# each tag MUST have a description
 	[ -n "$value" ] && return
 
+	echo -e "\n"
```
Comment (devimc) [inline scripts/configure-hypervisor.sh]: same here

File: scripts/configure-hypervisor.sh (hunk 1 above)
Comment (jodh-intel) [reply scripts/configure-hypervisor.sh]: Yeah, taking another look, this is just plain ugly so I've removed those changes.

File: scripts/configure-hypervisor.sh (hunk 2 above)
Comment (jodh-intel) [reply scripts/configure-hypervisor.sh]: Fixed.

**Comment by jodh-intel** [comment]
> would be better to squash all commits, I see one or two changes per commit

I'll let you and @grahamwhaley fight it out (he likes commits *a lot* :)

**Comment by egernst** [comment]
commits are free if it helps provide granularity/rationale.  I'm fine either way with this.

**Comment by grahamwhaley** [comment]
needs another ack from one of @kata-containers/packaging I think...

**Comment by jcvenegas** [comment]
lgtm

**Pull request merged**